### ISSUE TITLE
FR: Loading indication

### ISSUE THREAD

**Issue by tjapro** [sonnyp/Tangram#190]
I guess, from the visuals only, that a loading indication will be a good adition, like a line or a bar filling the top bar.

Note: this is a just an idea; can be pospotened.

**Comment by mediapinta** [comment]
Yes, sometimes I don't know if it friezed or it is just loading.

**Comment by sonnyp** [comment]
In the tab view, the reload button turns into a stop button when the page is loading.

In the overview there is a loading icon when the page is loading

We're missing one in the "new tab" view which probably needs a refresh anyway.
